option optimize_for = LITE_RUNTIME;

message MoneyTreeRequest
{
 required uint32 type = 1;
}
